### PR TITLE
[zh] Sync Izzy blog #14710 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync Izzy blog #14710 into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/blog/2024/istio-phippy/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+{{< image width="75%"
+    link="./phippy_and_family.png"
+    alt="Phippy 及其家族"
+    >}}
+
+Izzy 不仅代表了来自不同公司的 Istio 维护人员的辛勤工作和想象力，
+还将帮助我们以简单的方式阐释服务网格和 Istio 新 Ambient 模式的概念。
+请持续关注，我们将构建图文并茂的指南，
+Izzy 将以孩子可以理解的方式揭开 Istio 和服务网格的神秘面纱！
+下次当您为那些不了解您的背景知识的人分析这些概念时，使用 Izzy 如何？
```
Comment (hanxiaop) [inline content/zh/blog/2024/istio-phippy/index.md:37]:
```suggestion
下次当您为那些不了解您的背景知识的人解释这些概念时，使用 Izzy 如何？
```

File: content/zh/blog/2024/istio-phippy/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+Istio 最初由 Google 和 IBM 开发，基于 Lyft 的 Envoy 项目构建。
+该项目目前拥有来自超过 16 家公司的维护者，
+其中包括许多全球最大的网络供应商和云组织。
+Istio 提供零信任网络、策略执行、流量管理、负载平衡和监控，
+无需重写应用程序。
```
Comment (hanxiaop) [inline content/zh/blog/2024/istio-phippy/index.md:43]:
```suggestion
而无需重写应用程序。
```

**Review by hanxiaop — APPROVED** [review]
some nits

**Review by zirain — APPROVED** [review]

**Pull request merged**